test/functional/pdo_sqlsrv/PDO38_FetchBound.phpt

```diff
@@ -40,72 +34,78 @@ function Fetch()
     $stmt2 = PrepareQuery($conn1, "SELECT COUNT(id) FROM [$tableName]");
     $stmt2->execute();
     $num = $stmt2->fetchColumn();
-    echo 'There are ' . $num . " rows in the table.\n";
+    echo "There are $num rows in the table.\n";
     $stmt2->closeCursor();
 
     // Insert using named parameters
-    $stmt1 = PrepareQuery($conn1, "INSERT INTO [$tableName] VALUES(:first, :second, :third)");
-    foreach ($data as $row)
-    {
+    $stmt1 = $conn1->prepare("INSERT INTO [$tableName] VALUES(:first, :second, :third)");
+    foreach ($data as $row) {
         $stmt1->execute(array(':first'=>($row[0] + 5), ':second'=>$row[1], ':third'=>$row[2]));
     }
     unset($stmt1);
 
     $stmt2->execute();
     $num = $stmt2->fetchColumn();
-    echo 'There are ' . $num . " rows in the table.\n";
+    echo "There are $num rows in the table.\n";
 
-    $dataCols = "idx, txt";
-    CreateTableEx($conn1, $tableName, "idx int NOT NULL PRIMARY KEY, txt VARCHAR(20)", null);
-    InsertRowEx($conn1, $tableName, $dataCols, "0, 'String0'", null);
-    InsertRowEx($conn1, $tableName, $dataCols, "1, 'String1'", null);
-    InsertRowEx($conn1, $tableName, $dataCols, "2, 'String2'", null);
+    createTable($conn1, $tableName, array(new ColumnMeta("int", "idx", "NOT NULL PRIMARY KEY"), "txt" => "varchar(20)"));
+    insertRow($conn1, $tableName, array("idx" => 0, "txt" => 'String0'));
+    insertRow($conn1, $tableName, array("idx" => 1, "txt" => 'String1'));
+    insertRow($conn1, $tableName, array("idx" => 2, "txt" => 'String2'));
 
     // Testing with prepared query
-    $stmt1 = PrepareQuery($conn1, "SELECT COUNT(idx) FROM [$tableName]");
-
+    $stmt1 = $conn1->prepare("SELECT COUNT(idx) FROM [$tableName]");
     $stmt1->execute();
     var_dump($stmt1->fetchColumn());
     unset($stmt1);
 
-    $stmt1 = PrepareQuery($conn1, "SELECT idx, txt FROM [$tableName] ORDER BY idx");
+    if (!isColEncrypted()) {
+        $stmt1 = PrepareQuery($conn1, "SELECT idx, txt FROM [$tableName] ORDER BY idx");
+    } else {
+        // ORDER BY does not work on encrypted columns
+        $stmt1 = PrepareQuery($conn1, "SELECT idx, txt FROM [$tableName]");
+    }
     $stmt1->execute();
     $data = $stmt1->fetchAll(PDO::FETCH_COLUMN|PDO::FETCH_UNIQUE);
+    if (isColEncrypted()) {
```

Comment:
Do not use $data here because it's confusing when it's used in the beginning for data insertion.